tests/test_examples.py

```diff
@@ -50,26 +48,6 @@
 from .utils import area_difference_desc, area_difference_vmec
 
 
-@pytest.mark.regression
-@pytest.mark.solve
-def test_SOLOVEV_vacuum(SOLOVEV_vac):
-    """Tests that the SOLOVEV vacuum example gives no rotational transform."""
-    eq = EquilibriaFamily.load(load_from=str(SOLOVEV_vac["desc_h5_path"]))[-1]
-    data = eq.compute("|J|")
-
-    np.testing.assert_allclose(data["iota"], 0, atol=1e-16)
```

Comment:
ignoring whether it's well defined, doesn't this still unit test that those ~10 compute functions for iota yield zero as expected for this geometry?

Reply:
^ I think I agree

Reply:
That functionality is already tested with `test_iota_components`